src/fee-accounts/containers/overview-container/account-overview-container.component.html

```diff
@@ -0,0 +1,42 @@
+<ng-container *ngIf="isAccountAvailableForOrg; else newDeb">
+    <div [ngClass]="{'govuk-form-group govuk-form-group--error' : oneOrMoreAccountInfoMissing}">
+        <span *ngIf="oneOrMoreAccountInfoMissing" id="fee-error" class="govuk-error-message">
+            <span class="govuk-visually-hidden">Error:</span> PBA number not found
+        </span>
+        <table class="{{'govuk-table ' + classes}}" role="presentation">
```

Comment:
how come we don't use ngClass here?

Reply:
Can I park this?

Reply:
yeah pardeep is reight needs to meet ng standards this is definitely not it

Reply:
yeah sorry, wasn't actually asking for a code change, more curiousity lol